src/igvfd/schemas/analysis_step.json

```diff
@@ -78,15 +83,20 @@
                     "barcode mapping generation": "The process of assigning barcode sequences to specific reads for identification and separation of samples.",
                     "base calling": "The conversion of raw sequencing signals into nucleotide sequences, determining the identity of each base in the reads.",
                     "counts normalization": "Adjusting raw read counts to account for biases in sequencing depth or composition.",
+                    "fastq concatenation": "The process of combining fastqs files from multiple lanes.",
                     "file format conversion": "The conversion of data files from one file format to another.",
                     "filtering": "The removal of low-quality reads or sequences based on specified criteria.",
+                    "fragment generation": "The process of generating fragments based on ATAC reads.",
                     "interaction calling": "Identifying and characterizing interactions between genomic elements.",
+                    "matrix generation": "The process of generating matrix file.",
                     "merging": "Combining multiple datasets or files into a single dataset.",
                     "peak calling": "The identification of regions in the genome where there is a significant enrichment of reads.",
+                    "read trimming": "The trimming of reads may include removing adapter sequences and trimming reads off low quality base pairs",
                     "quantification": "The measurement of gene expression or other features based on read counts.",
                     "signal generation": "The process of generating a continuous signal representation of aligned sequencing data.",
                     "signal normalization": "Adjusting the generated signal to remove systematic biases.",
                     "spatial feature detection": "The identification of spatially resolved features in genomic data, often used in spatial transcriptomics.",
+                    "UMI deduplication": "The process of removing UMI duplicates.",
```

Comment:
can we just add the full name UMI to the description? Also 'UMI quantification' is missing from the description, can you add it together here?

Reply:
yeap, i can. Added.